From 11e72a5143996d73eb47f75e6e1c58d0567c6870 Mon Sep 17 00:00:00 2001
From: Joe Conley <joe.conley@lairdtech.com>
Date: Fri, 5 Feb 2016 14:48:59 -0500
Subject: [PATCH] laird: ath6kl: remove unused __freq_is_specified function

This function is no longer used and no longer needed.

bug 8899
---
 drivers/net/wireless/ath/ath6kl/wmi.c | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/drivers/net/wireless/ath/ath6kl/wmi.c b/drivers/net/wireless/ath/ath6kl/wmi.c
index b3222cb..0fc7337 100644
--- a/drivers/net/wireless/ath/ath6kl/wmi.c
+++ b/drivers/net/wireless/ath/ath6kl/wmi.c
@@ -4250,18 +4250,6 @@ int ath6kl_wmi_control_rx(struct wmi *wmi, struct sk_buff *skb)
 	return ath6kl_wmi_proc_events(wmi, skb);
 }
 
-static bool __freq_is_specified(struct ath6kl *ar, const u16 freq)
-{
-	u32 i;
-	u32 n_channels = ar->laird.num_channels;
-
-	for(i = 0; i < n_channels; i++)
-		if(freq == ar->laird.channel_list[i])
-			return true;
-
-	return false;
-}
-
 int ath6kl_wmi_channel_params_cmd(struct wmi *wmi, u8 if_idx, u8 scan_param,
 			    u8 phy_mode, u8 num_channels, u16 *channel_list)
 {
-- 
1.9.1

